mm2src/coins/solana.rs

```diff
@@ -749,14 +749,14 @@ impl MarketCoinOps for SolanaCoin {
 
     fn min_trading_vol(&self) -> MmNumber { MmNumber::from("0.00777") }
 
+    fn should_burn_dex_fee(&self) -> bool { false }
```

Comment:
Same here and for spl

Reply:
fixed

Reply:
This is still not fixed, the comment is for solana and spl not sia.

Reply:
sorry, now fixed in 1e792165601da7015a53073c185321a4a558cf00